src/utils/ProfileUtils.ts

```diff
@@ -16,6 +16,8 @@ import ZipProvider from "../providers/generic/zip/ZipProvider";
 import ProfileInstallerProvider from "../providers/ror2/installing/ProfileInstallerProvider";
 import * as PackageDb from '../r2mm/manager/PackageDexieStore';
 import ProfileModList from "../r2mm/mods/ProfileModList";
+import ConflictManagementProvider from "src/providers/generic/installing/ConflictManagementProvider";
+import {Store} from "vuex";
```

Comment:
Nitpick: if we want to keep the import order logic, this should be located before yaml as it's a 3rd party import.